calico/common.py

```diff
@@ -46,6 +46,9 @@
 # string used by the logger.
 SYSLOG_FORMAT_STRING = '{excname}[%(process)s]: %(module)s@%(lineno)d %(message)s'
 
+KERNEL_PROTOCOLS = set(["tcp", "udp", "icmp", "icmpv6", "sctp", "udplite",
+                        "sctp"])
```

Comment:
I realise we all love sctp, but it probably doesn't need to be in the list twice.